apps/coscrad-frontend/src/components/resources/songs/test-utils/build-dummy-songs.ts

```diff
@@ -0,0 +1,23 @@
+import { ISongViewModel } from '@coscrad/api-interfaces';
+
+export const buildDummySongs = (): ISongViewModel[] => [
+    {
+        id: '12',
+        title: 'ba ba ba',
+        titleEnglish: 'cool song',
+        lyrics: 'go sing ba, ba, ba, ba, ba, ba, ba, ba ,ba',
```

Comment:
Not top priority : ) but there's a comma - space inversion in both the lyrics...

Reply:
Great eye for detail! I know this is just dummy data, but I don't want it to look sloppy.

Reply:
fixed